src/openms/include/OpenMS/FORMAT/DATAACCESS/CsiFingerIdMzTabWriter.h

```diff
@@ -83,6 +83,9 @@ namespace OpenMS
 
           struct CsiAdapterIdentification
```

Comment:
Perhaps introduce a base struct for the current structs `CSIAdapterIdentification` and `SiriusAdapterIdentification` and there declare the fields `mz, rt, native_id, scan_index, scan_number, feature_id`?  Then the derived classes would only add  the vector with the respective hits.

Perhaps also explain the why the id is called "native".

Reply:
native_id -> The nativeID is usually a unique reference for a spectrum in a file based on the vendor. e.g. "sample=1 period=1 cycle=2177 experiment=5"

Reply:
You are right this could be done using a base struc/class, but I think this is not really necessary at this point.